### PR TITLE
Updates docs

### LINKED ISSUE #159: server installation steps are missing etcd config part
Installation doc doesn't have etcd configuration steps.

### LINKED ISSUE #299: document db.DB interface
The DB interface and all its methods should be documented.

At the same time, its implementations shouldn't receive any special documentation besides `Method implements DB.Method`, and they shouldn't receive any important documentation such as `// Pass count = -1 to get all elements starting from the provided index`, as that of information belongs to the interface documentation.

### PR DESCRIPTION
**Pull request by chrisvdg**
Updates and corrects docs

Closes #159
Closes #299

### CODE REVIEW

**Comment by chrisvdg** [comment]
If there are other parts of the docs that need to be updated, you can let me know!

**Comment by zaibon** [comment]
There is also #216 which relates to documentation. But doesn't have to be in this PR

**Comment by chrisvdg** [comment]
@zaibon, I can add it, if only I know how to do it :D 
Anything I can look at for inspiration?

File: client/README.md [hunk 1]
```diff
@@ -110,12 +121,11 @@ It can be used for example to reconstruct the stored sequential data.
 
 ## Using 0-stor client examples:
 
-
 In below example, when we store the data, the data will be processed as follow:
 plain data -> compress -> encrypt -> distribution/erasure encoding (which send to 0-stor server and write metadata)
 
 When we get the data from 0-stor, the reverse process will happen:
-distribution/erasure decoding (which read metdata & Get data from 0-stor) -> decrypt -> decompress -> plain data.
+distribution/erasure decoding (which read metadata & Get data from 0-stor) -> decrypt -> decompress -> plain data.
```
Comment (GlenDC) [inline client/README.md]: reads*

**Pull request merged**